ts/parts/Tooltip.ts

```diff
@@ -1134,6 +1147,45 @@ class Tooltip {
             options.outside,
             Boolean(chart.scrollablePixelsX || chart.scrollablePixelsY)
         );
+
+        this.pointerEvents = options.style?.pointerEvents || (options.stickOnContact ? 'auto' : 'none');
+    }
+
+    /**
+     * Returns true, if pointer event is in contact with stick tooltip.
+     *
+     * @private
+     * @function Highcharts.Tooltip#isStickOnContact
+     *
+     * @param {Highcharts.PointerEventObject} e
+     * Pointer event to test.
+     *
+     * @return {boolean}
+     * True, if pointer is in contact with sticky tooltip.
+     */
+    public isStickyOnContact(e: Highcharts.PointerEventObject): boolean {
```

Comment:
Now that we have refactored the `stickOnContact` logic to work with real elements and mouseover/out events, do we need this computational hit testing at all? We have the following situations:

* Mouse from point to tooltip tracker => Tooltip sticks. Events detect it.
* Mouse from point to anything else => Tooltip hides. Events detect it.
* Mouse from tooltip tracker to own points and series (tooltip is aware of its point) => Tooltip sticks. Events detect it.
* Mouse from tooltip to anything else => Tooltip hides because related target is not its own point or series.

I can't see a situation where we need hit testing?

Reply:
I need this in sankey at the moment, but maybe there is an other way to solve the issue with the link hover handling.

Reply:
I partially fixed it, but it still jumps sometimes into the center link for unknown reason.